sled-agent/src/bootstrap/rack_ops.rs

```diff
@@ -164,9 +167,12 @@ impl RssAccess {
         }
     }
 
-    pub(super) fn start_initializing(
+    pub(crate) fn start_initializing(
         &self,
-        agent: &Arc<Agent>,
+        parent_log: &Logger,
+        global_zone_bootstrap_ip: Ipv6Addr,
+        storage_resources: &StorageResources,
+        bootstore_node_handle: &bootstore::NodeHandle,
```

Comment:
Makes sense, seems like the crux of the refactor is pulling these values out of the agent itself.